Cargo.toml

```diff
@@ -9,4 +9,10 @@ members = [
 ]
 
 [profile.release]
-panic = "unwind"
\ No newline at end of file
+codegen-units = 1
+lto = true
+opt-level = 3
+panic = "unwind"
```

Comment:
I think we should have a `production` profile with the optimizations and keep the old release profile as is.

Reply:
Yeah, I thought about it. The binary will generated under `target/production`, and some path need to be adapted.

Reply:
We could add `--features=fast-runtime` to a new `profile.testnet` when https://github.com/Manta-Network/Manta/pull/383 is merged (it's ready to)

Reply:
> We could add `--features=fast-runtime` to a new `profile.dev` and maybe `profile.release` to make testnets deploy with fast times by default when #383 is merged (it's ready to)

cargo profile cannot include features, only rustc flags.

Reply:
This looks resolved now, no ?

Reply:
Please resolve the conversations when you're done.